timesketch/frontend-ng/src/components/Explore/UnfurlDialog.vue

```diff
@@ -12,66 +12,57 @@ limitations under the License.
 -->
 <template>
   <v-card>
-    <v-card-title>
-      <v-img :src="getUnfurlLogo" max-height="32" contain class="mb-2"></v-img>
-    </v-card-title>
+    <v-toolbar flat color="transparent">
+      <v-img position="left" :src="getUnfurlLogo" max-height="24" contain class="ml-2"></v-img>
+    </v-toolbar>
+
     <v-card-subtitle class="pt-1">
-      <span><b>Input:</b> {{ url }}</span>
+      <div class="mb-2"><b>Input:</b> {{ url }}</div>
+      <div v-if="nodeContext">
```

Comment:
```suggestion
      <div>
```

I would highly recommend to keep the default call to action `Select a node in the graph below to get more information.` visible when the dialog opens the first time.

My user tests have shown that the call to action is needed for users to click the graph nodes and to get the best experience and information from unfurl.

Reply:
fair point, I'll add it back